Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheetCell.swift

```diff
@@ -87,12 +87,11 @@ class PhotonActionSheetCell: UITableViewCell {
             mainView.addSubview(foreground)
             setOn(false)
         }
-
+        
```

Comment:
nit: revert this spacing change

Reply:
Hello @dnarcese, suggested formatting changes are in this 655d5d955f31d9f8ccd23c0cad89c50d48cbedb8 commit. Thank you :)